app/rdf/extended-cube.ts

```diff
@@ -0,0 +1,29 @@
+import clownface, { AnyPointer } from "clownface";
+import { Cube, CubeOptions } from "rdf-cube-view-query";
+import rdf from "rdf-ext";
+import DatasetExt from "rdf-ext/lib/Dataset";
+
+export class ExtendedCube extends Cube {
```

Comment:
Cna you add a docstring describing the general goal ? So that when we see a method that should be there, we can be sure that this class is the right one ?